float32 linearVx
float32 linearVy
float32 angularVz